rna/serializers.py

```diff
@@ -22,20 +22,21 @@ def get_pk_field(self, model_field):
         Returns a default instance of the pk field, unlike
         the parent class, which omits the pk field.
         """
-        return self.get_field(model_field)
+        return self.get_fields().get(model_field)
 
 
 class UnmodifiedTimestampSerializer(serializers.ModelSerializer):
```

Comment:
@jgmize need to make this work again I think. Not sure exactly what was going on here though and whether it's necessary in new DRF. Thoughts?